### PR TITLE
[DO NOT MERGE] EOS-16666:S3: moved s3cipher from s3backgrounddelete to s3cortxutils

### PR DESCRIPTION
**Pull request by vimalraghwani20**
Jenkins job:
http://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/S3server-7.8.2003/542/console

### CODE REVIEW

File: s3cortxutils/s3cipher/s3cipher/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
```
Comment (amitwac2608) [inline s3cortxutils/s3cipher/s3cipher/README.md:1]: this is md file, '#' will be interpreted differently here. Put the copyright header as plain text, no need of "#' or something else

**Review by amitwac2608 — COMMENTED** [review]
looks good. Make sure you fix all codacy issues.

Also in case RE team is not ready with "mini-provisioner" pre-merge jenkins job, then we need to do manual testing before merge, so that your changes do not break mini-provisioning flow.

File: s3cortxutils/s3cipher/s3cipher/README.md (hunk 1 above)
Comment (vimalraghwani20) [reply s3cortxutils/s3cipher/s3cipher/README.md]: fixed

File: s3cortxutils/s3cipher/s3cipher/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+
+Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (nileshgovande) [inline s3cortxutils/s3cipher/s3cipher/README.md:4]: Write use cases

**Review by nileshgovande — APPROVED** [review]

File: s3cortxutils/s3cipher/s3cipher/README.md (hunk 2 above)
Comment (vimalraghwani20) [reply s3cortxutils/s3cipher/s3cipher/README.md:4]: fixed

**Comment by gowthamchinna** [comment]
run mini-provisioning

**Comment by vimalraghwani20** [comment]
Closing as changes has been merged in through other PR.

**Pull request closed without merging**